framework/validators/UniqueValidator.php

```diff
@@ -69,6 +78,9 @@ public function init()
         if ($this->message === null) {
             $this->message = Yii::t('yii', '{attribute} "{value}" has already been taken.');
         }
+	    if ($this->comboNotUnique === null) {
+		    $this->comboNotUnique = Yii::t('yii', 'Combination [{valueCombo}] of [{attributeCombo}] has already been taken.');
```

Comment:
just add it to the english translation, then the translators (including myself) will use that to translate it to other languages.